### ISSUE TITLE
Deployment failed - External links valid?

### ISSUE THREAD

**Issue by svbfromnl** [f5devcentral/f5-azure-saca#116]
xxxx-bigip0/start failed to deploy. Attempting a commercial deployment to non-gov cloud space directly from GitHub page.

Azure Status message:
```
{
    "status": "Failed",
    "error": {
        "code": "ResourceDeploymentFailure",
        "message": "The resource operation completed with terminal provisioning state 'Failed'.",
        "details": [
            {
                "code": "VMExtensionProvisioningError",
                "message": "VM has reported a failure when processing extension 'start'. Error message: \"Enable failed: processing file downloads failed: failed to download file[6]: failed to download file: unexpected status code: actual=404 expected=200\"\r\n\r\nMore information on troubleshooting is available at https://aka.ms/VMExtensionCSELinuxTroubleshoot "
            }
        ]
    }
}
```

bigip2 failed with a more verbose error:
```
{
    "status": "Failed",
    "error": {
        "code": "ResourceDeploymentFailure",
        "message": "The resource operation completed with terminal provisioning state 'Failed'.",
        "details": [
            {
                "code": "VMExtensionProvisioningError",
                "message": "VM has reported a failure when processing extension 'start'. Error message: \"Enable failed: failed to execute command: command terminated with exit status=1\n[stdout]\nnfo:  Installing package at path: /var/config/rest/downloads/f5-appsvcs-3.16.0-6.noarch.rpm \n2020-07-20T22:35:27.190Z info:  Saving config. \n2020-07-20T22:35:31.540Z info:  Waiting for device to be active. \n2020-07-20T22:35:31.979Z info:  Device onboard complete. \n2020-07-20T22:35:32.383Z info:  Onboard finished. \n2020-07-20T22:35:32.570Z info:  /config/cloud/azure/node_modules/@f5devcentral/f5-cloud-libs/scripts/network.js called with /usr/bin/f5-rest-node /config/cloud/azure/node_modules/@f5devcentral/f5-cloud-libs/scripts/network.js --output /var/log/cloud/azure/network.log --wait-for ONBOARD_DONE --host 192.168.1.9 --port 443 -u svc_user --password-url file:///config/cloud/.passwd --password-encrypted --default-gw 192.168.3.1 --vlan name:external,nic:1.1 --vlan name:internal,nic:1.2 --self-ip name:self_2nic,address:192.168.2.5,vlan:external --self-ip name:self_3nic,address:192.168.3.5,vlan:internal --log-level info \n2020-07-20T22:35:32.579Z info:  Waiting for ONBOARD_DONE \n2020-07-20T22:35:33.581Z info:  Network setup starting. \n2020-07-20T22:35:33.583Z info:  Initializing BIG-IP. \n2020-07-20T22:35:33.598Z info:  This is a BIG-IP \n2020-07-20T22:35:35.572Z info:  Waiting for device to be ready. \n2020-07-20T22:35:36.078Z info:  Waiting for BIG-IP to be ready. \n2020-07-20T22:35:36.491Z info:  BIG-IP is ready. \n2020-07-20T22:35:36.492Z info:  Creating vlan external on interface 1.1   untagged \n2020-07-20T22:35:36.724Z info:  Creating vlan internal on interface 1.2   untagged \n2020-07-20T22:35:36.905Z info:  Creating self IP self_2nic with address 192.168.2.5/24 on vlan external allowing default \n2020-07-20T22:35:37.034Z info:  Creating self IP self_3nic with address 192.168.3.5/24 on vlan internal allowing default \n2020-07-20T22:35:37.084Z info:  Setting default gateway 192.168.3.1 \n2020-07-20T22:35:37.118Z info:  Saving config. \n2020-07-20T22:35:41.111Z info:  BIG-IP network setup complete. \n2020-07-20T22:35:41.112Z info:  Network setup finished. \n2020-07-20T22:35:42.628Z info:  /config/cloud/azure/node_modules/@f5devcentral/f5-cloud-libs/scripts/cluster.js called with /usr/bin/f5-rest-node /config/cloud/azure/node_modules/@f5devcentral/f5-cloud-libs/scripts/cluster.js --output /var/log/cloud/azure/cluster.log --log-level info --host 192.168.1.9 --port 443 -u svc_user --password-url file:///config/cloud/.passwd --password-encrypted --config-sync-ip 192.168.3.5 --join-group --device-group Sync --sync --remote-host 192.168.1.4 --remote-user svc_user --remote-password-url file:///config/cloud/.passwd \n2020-07-20T22:35:42.638Z info:  Cluster starting. \n2020-07-20T22:35:42.640Z info:  Initializing BIG-IP. \n2020-07-20T22:35:42.654Z info:  This is a BIG-IP \n2020-07-20T22:35:44.666Z info:  Waiting for device to be ready. \n2020-07-20T22:35:45.145Z info:  Waiting for BIG-IP to be ready. \n2020-07-20T22:35:45.562Z info:  BIG-IP is ready. \n2020-07-20T22:35:45.563Z info:  Setting config sync ip. \n2020-07-20T22:35:46.447Z info:  Joining group. \n2020-07-20T22:35:46.461Z info:  This is a BIG-IP \n2020-07-20T22:35:48.475Z info:  Waiting for device to be ready. \n2020-07-20T22:53:49.829Z info:  Device initialization failed tryUntil: max tries reached: tryUntil: max tries reached: <!DOCTYPE HTML PUBLIC \"-//IETF//DTD HTML 2.0//EN\">\n<html><head>\n<title>401 Unauthorized</title>\n</head><body>\n<h1>Unauthorized</h1>\n<p>This server could not verify that you\nare authorized to access the document\nrequested.  Either you supplied the wrong\ncredentials (e.g., bad password), or your\nbrowser doesn't understand how to supply\nthe credentials required.</p>\n</body></html>\n \n2020-07-20T22:53:49.830Z info:  join cluster failed: tryUntil: max tries reached: tryUntil: max tries reached: <!DOCTYPE HTML PUBLIC \"-//IETF//DTD HTML 2.0//EN\">\n<html><head>\n<title>401 Unauthorized</title>\n</head><body>\n<h1>Unauthorized</h1>\n<p>This server could not verify that you\nare authorized to access the document\nrequested.  Either you supplied the wrong\ncredentials (e.g., bad password), or your\nbrowser doesn't understand how to supply\nthe credentials required.</p>\n</body></html>\n \n\n[stderr]\n01020036:3: The requested system device (/Common/bigip1.eastus.cloudapp.azure.com) was not found.\n{\n  \"level\": \"error\",\n  \"message\": \"Cluster failed: tryUntil: max tries reached: tryUntil: max tries reached: <!DOCTYPE HTML PUBLIC \\\"-//IETF//DTD HTML 2.0//EN\\\">\\n<html><head>\\n<title>401 Unauthorized</title>\\n</head><body>\\n<h1>Unauthorized</h1>\\n<p>This server could not verify that you\\nare authorized to access the document\\nrequested.  Either you supplied the wrong\\ncredentials (e.g., bad password), or your\\nbrowser doesn't understand how to supply\\nthe credentials required.</p>\\n</body></html>\\n\",\n  \"label\": \"lib/bigIpOnboard.js\",\n  \"timestamp\": \"2020-07-20T22:53:49.832Z\"\n}\n2020-07-20T22:53:49.833Z error:  Cluster failed: tryUntil: max tries reached: tryUntil: max tries reached: <!DOCTYPE HTML PUBLIC \"-//IETF//DTD HTML 2.0//EN\">\n<html><head>\n<title>401 Unauthorized</title>\n</head><body>\n<h1>Unauthorized</h1>\n<p>This server could not verify that you\nare authorized to access the document\nrequested.  Either you supplied the wrong\ncredentials (e.g., bad password), or your\nbrowser doesn't understand how to supply\nthe credentials required.</p>\n</body></html>\n \n\"\r\n\r\nMore information on troubleshooting is available at https://aka.ms/VMExtensionCSELinuxTroubleshoot "
            }
        ]
    }
}
```

Are all the external links correct?

**Comment by svbfromnl** [comment]
It looks like this link is incorrect:
https://github.com/f5devcentral/f5-azure-saca/blob/1f5f6ded16226a397c2672e1e75cafbf6df97c37/SACAv2/3NIC_1Tier_HA/payg/azureDeploy.json#L1657

**Comment by svbfromnl** [comment]
Looks like there is another one pointing to your personal repo @Mikej81 
https://github.com/f5devcentral/f5-azure-saca/blob/1f5f6ded16226a397c2672e1e75cafbf6df97c37/SACAv2/3NIC_1Tier_HA/payg/azureDeploy.json#L255

**Comment by Mikej81** [comment]
ASM policy references have been moved out of personal repos and should point to [https://github.com/f5devcentral/f5-asm-policy-templates](https://github.com/f5devcentral/f5-asm-policy-templates).  Will update here as soon as time permits.

**Comment by svbfromnl** [comment]
> ASM policy references have been moved out of personal repos and should point to https://github.com/f5devcentral/f5-asm-policy-templates. Will update here as soon as time permits.

Thanks Mike

**Comment by Mikej81** [comment]
fixed in #117
